### PR TITLE
New package: Acronis.CyberProtectHomeOffice version 26.1.1.39703

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57309)

### CODE REVIEW

File: manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Acronis.CyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
```
Comment (OfficialEsco) [inline manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml:5]:
```suggestion
PackageVersion: 26.1.39703
```

File: manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Acronis.CyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
+PackageLocale: en-US
+Publisher: acronis
```
Comment (OfficialEsco) [inline manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.locale.en-US.yaml:7]:
```suggestion
Publisher: Acronis
```

File: manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Acronis.CyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
```
Comment (OfficialEsco) [inline manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 26.1.39703
```

File: manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Acronis.CyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
```
Comment (OfficialEsco) [inline manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.yaml:5]:
```suggestion
PackageVersion: 26.1.39703
```

File: manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: -silent_installation
+  SilentWithProgress: -start_installation
```
Comment (SpecterShell) [inline manifests/a/Acronis/CyberProtectHomeOffice/26.1.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml:10]:
```suggestion
  SilentWithProgress: -start_installation
  Log: -log:"<LOGPATH>"
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/a/Acronis/CyberProtectHomeOffice/26.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Acronis.CyberProtectHomeOffice
+PackageVersion: 26.1.39703
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: -silent_installation
+  SilentWithProgress: -start_installation
```
Comment (OfficialEsco) [inline manifests/a/Acronis/CyberProtectHomeOffice/26.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml:10]:
```suggestion
  SilentWithProgress: -silent_installation #-start_installation
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Since it will be flagged as unattended we need to use silent for both

File: manifests/a/Acronis/CyberProtectHomeOffice/26.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+InstallerType: exe
+InstallerSwitches:
+  Silent: -silent_installation
+  SilentWithProgress: -silent_installation #-start_installation
+  Log: -log:"<LOGPATH>"
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.acronis.com/u/AcronisCyberProtectHomeOffice.exe
+  InstallerSha256: 136F1D43F363097B8F85FC06B56EA045D57610AAD43268F129D388C551D26514
```
Comment (Trenly) [inline manifests/a/Acronis/CyberProtectHomeOffice/26.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml:16]: This package may require AppsAndFeaturesEntries to be able to upgrade -
```suggestion
  InstallerSha256: 136F1D43F363097B8F85FC06B56EA045D57610AAD43268F129D388C551D26514
  Scope: machine
  AppsAndFeaturesEntries: 
  - DisplayName: Acronis Cyber Protect Home Office
    DisplayVersion: 26.1.39703
    InstallerType: msi
    ProductCode: '{E1FEA3C4-5790-465E-ADDD-6FF1E9FDCAB4}'
  - DisplayName: Acronis Cyber Protect Home Office
    DisplayVersion: 26.1.39703
    InstallerType: exe
    ProductCode: '{E1FEA3C4-5790-465E-ADDD-6FF1E9FDCAB4}Visible'
  - DisplayName: Acronis Drivers
    DisplayVersion: 26.1.39703
    InstallerType: msi
    ProductCode: '{C5293FA2-D7C1-4445-9DC9-3E34349D59AF}'
```

**Review by OfficialEsco — APPROVED** [review]

**Pull request closed without merging**